### ISSUE TITLE
Exempt items from clipper

### ISSUE THREAD

**Issue by ThomasvHoutum** [ThatOpen/engine_components#544]
### Description 📝

I am using the [Plans](https://docs.thatopen.com/Tutorials/Components/Front/Plans) feature to display some floorplans of the IFC model.

What I would like to achieve is to load a mesh on top of the plan.

Currently this is not possible, as the clipper used for the plans is set at a certain height and anything placed above it will disappear. So placing the mesh above the model is not going to work.
Here is a screenshot:
![image](https://github.com/user-attachments/assets/5c2f7495-e906-426e-aef7-4eaa085d26be)

### Suggested solution 💡

If possible, if the clipper would get a feature where we could specify what items should get clipped, and which shouldn't. With that we could make it so when the Plans component generates the sections we would only pass through the model object to the clipper, so anything else added to the world's scene would not get clipped.

### Validations ✅

- [X] Read the [docs](https://docs.thatopen.com/intro).
- [X] Check that there isn't [already an issue](https://github.com/ThatOpen/engine_components/issues) that requests the same feature to avoid creating a duplicate.

**Comment by agviegas** [comment]
I don't think this is possible at the moment because the postproduction effects that we use rely on global clipping 😵‍💫 As a caveat, you can:
- Create objects below the clipping plane (maybe add a materials so that it's always visible above other objects)
- Set up your own three.js scene / components to work with local clipping
Hope this helps a bit!
